src/hooks/useKeyLongPressed.ts

```diff
@@ -23,7 +23,7 @@ export const useKeyLongPressed = (): [string] => {
       document.removeEventListener('keydown', handleKeyDown);
       document.removeEventListener('keyup', handleKeyUp);
     };
-  },[]);
+  }, []);
 
   return [keyLongPressed];
 };
```

Comment:
please fix all the lint fix related issue in the file to maintain minimum delta

Reply:
fixed

Reply:
fixed